pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -279,6 +279,9 @@ type GameServerAllocationStatus struct {
 	Ports          []agonesv1.GameServerStatusPort `json:"ports,omitempty"`
 	Address        string                          `json:"address,omitempty"`
 	NodeName       string                          `json:"nodeName,omitempty"`
+	// If the allocation is from a remote cluster, Source is the endpoint of the remote agones-allocator.
+	// Otherwise, Source is "local"
+	Source         string                          `json:"source"`  //nolint:goimports
```

Comment:
//nolint:goimports

What check is it to close?

Reply:
I think we should use gofmt formatting, even though the formatting may not be ideal, but he can become a standard, and if other participants modify the problem subsequently, he will eventually become gofmt format, otherwise the participants need to control the aspect of the modification.

So I don't think we should have to force to align the format above.

Thanks

Reply:
Sounds fair. Also I found the [old discussion](https://github.com/golang/go/issues/29784) around the issue. So `gofmt` behaves as designed, though the behavior doesn't properly fit all the use cases.